### ISSUE TITLE
Contacts: Request message is not added to the chat on acceptance when re-adding

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#10522]
# Bug Report

## Description

Split from and dependant on https://github.com/status-im/status-desktop/pull/10336

Alice removes Bob as a contact
Alice sends a contact request to Bob with the invite message of Hello again
Bob views the contact request and message in the Activity Centre
Bob accepts the contact request but the invite message is not added to the chat like the first time a contact is accepted

### Additional Information

- Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/aarch64/job/package/job/PR-10336/20/
- Operating System: Mac

**Comment by jrainville** [comment]
@MishkaRogachev is this a duplicate of one of the issues you are already working on. I feel like you are/were working on this lately

**Comment by MishkaRogachev** [comment]
Yes, this one should also be fixed here: https://github.com/status-im/status-desktop/pull/10613
